src/layouts/secondary-navigation-layout/SecondaryNavigationLayout.module.scss

```diff
@@ -0,0 +1,97 @@
+@import '~terra-breakpoints/lib/media-queries';
+@import './clinical-lowlight-theme/SecondaryNavigationLayout.module';
+@import './orion-fusion-theme/SecondaryNavigationLayout.module';
+
+:local {
+  .sidebar-container {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    outline: none;
+    padding-right: 2px;
+    position: relative;
+    width: 100%;
+    z-index: 0;
+
+    .header-container {
+      flex: 0 0 auto;
+      z-index: 0;
+    }
+
+    .content {
+      flex: 1 1 auto;
+      height: 100%;
+      min-height: 50px;
+      overflow: hidden;
+      position: relative;
+      width: 100%;
+      z-index: 1;
+    }
+  }
+
+  .side-nav-container {
+    align-items: flex-start;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    height: 100%;
+    justify-content: stretch;
+    overflow: hidden;
+    position: relative;
+  }
+
+  .side-nav-sidebar {
+    background-color: var(--terra-dev-site-secondary-navigation-layout-side-nav-sidebar-background-color, #eff0ef);
+    box-shadow: inset -3px 0px 0px -2px#fefffe, inset -4px 0px 0px -2px #C7CACB;
+    display: none;
+    flex: 0 0 auto;
+    height: 100%;
+    width: 280px;
+
+    &.visible {
+      display: block;
+    }
+
+    &.overlay {
+      position: absolute;
+      width: calc(100% - 50px);
+      left: 0;
+      top: 0;
+      bottom: 0;
+      z-index: 6;
+
+      @include terra-mq-medium-up {
+        width: 200px;
+      }
+    }
+  }
+
+  .side-nav-body {
+    align-items: flex-start;
+    display: flex;
+    flex: 1 1 auto;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    height: 100%;
+    justify-content: stretch;
+    overflow: hidden;
+  }
+
+  .page-body {
+    flex: 1 1 320px;
+    height: 100%;
+    overflow: hidden;
+  }
+
+  .side-nav-resize-overlay {
+    bottom: 0;
+    cursor: col-resize;
+    display: none;
+    left: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+    z-index: 5; // TODO 5 is very important for safari, validate final value for this
```

Comment:
You could remove this class as well (and not have to worry about the TODO, hurray)